docs/example-scenario/apps/ecommerce-scenario.md

```diff
@@ -0,0 +1,152 @@
+---
+title: E-Commerce on Azure Platform as a Service
+description: Proven scenario of hosting an e-commerce site on Azure
+author: masonch
+ms.date: 7/13/18
+---
+# E-Commerce on Azure Platform as a Service
+
+Azure Platform-as-a-Service (PaaS) enables you to deploy enterprise grade e-commerce applications, and lets you adapt to the size and seasonality of your business. When demand for your products or services takes off — predictably or unpredictably — you can be prepared to handle more customers and more transactions automatically. Additionally, take advantage of cloud economics by paying only for the capacity you use. In short, focus on your sales and leave the infrastructure management to your cloud provider.
```

Comment:
Starts with the solution, not with the workload and sounds like marketing fluff.  How about something like:

This example scenario walks you through an implementation of an ecommerce front end using Azure Platform-as-a-Service (PaaS) tools. Many ecommerce websites face seasonality and traffic variability over time. When demand for your products or services takes off, whether predictably or unpredictably, using PaaS tools will allow you to handle more customers and more transactions automatically. Additionally, this scenario takes advantage of cloud economics by paying only for the capacity you use.